OverTool/List/ListAbilityInfo.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using CASCExplorer;
+using OWLib;
+using OWLib.Types.STUD;
+
+namespace OverTool.List {
+    public class ListAbilityInfo : IOvertool {
+        public string Title => "List Abilities";
+        public char Opt => ' ';
```

Comment:
As of ba519647b8b1bee39b07275b8827ea5331514098, all "no" opts are `'\0'` or `(char) 0`

will merge when complete